modules/openapi-generator/src/main/resources/dart2/class.mustache

```diff
@@ -75,6 +73,8 @@ class {{classname}} {
       {{/isDateTime}}
     {{/vars}}
     };
+    json.removeWhere((key, value) => value == null);
```

Comment:
In the future, we want to support nullable so this will not work well with nullable property.

Ref: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#fixed-fields-20

Reply:
yes, I agree. so I guess this should be an option, like -DomitNullFields?

Reply:
Are you free for a quick chat this week? Please ping me via https://gitter.im (ID: wing328) when you've time.